src/packages/files/files.ts

```diff
@@ -89,6 +90,60 @@ export default (app: Express) => {
     }
   });
 
+  /**
+   * @swagger
+   * /file/{id}:
+   *   get:
+   *     parameters:
+   *       - description: ID of the file
+   *         in: path
+   *         name: id
+   *         required: true
+   *         schema:
+   *           type: number
+   *     produces:
+   *       - application/json
+   *     responses:
+   *       200:
+   *         description: Fetches the file by id
+   *     summary: Get the file with the associated ID.
+   *     tags:
+   *       - Files
+   */
+  app.get('/file/:id', filesLimiter, auth, async (req, res) => {
+    const { params } = req;
+    const { id } = params;
+
+    try {
+      const { rows, rowCount } = await db.query(
+        `
+        select *
+        from merlin.uploaded_file
+        where id = $1
+        and deleted_date is null;
+      `,
+        [id],
+      );
+
+      if (rowCount && rowCount > 0) {
```

Comment:
Blocking: if this `if` condition isn't met, the method ends without returning _anything_. I'd recommend the following elses:

```
else if (!rowCount) {
 //return 404 - file not found
}
else {
 // either throw an error or return a 500
}